types/react-simple-maps/index.d.ts

```diff
@@ -88,6 +89,38 @@ export interface ZoomableGroupProps extends React.SVGAttributes<SVGGElement> {
     translateExtent?: [[number, number], [number, number]] | undefined;
 }
 
+interface useZoomPanProps {
```

Comment:
Is this interface intentionally lowercased? Typically these are uppercase (note the previous examples).

Reply:
No not intentional, I will update it